extensions/wikia/Email/EmailController.class.php

```diff
@@ -34,6 +38,7 @@ public function init() {
 			$this->currentUser = $this->findUserFromRequest( 'currentUser', $this->wg->User );
 			$this->targetUser = $this->findUserFromRequest( 'targetUser', $this->wg->User );
 			$this->test = $this->getRequest()->getVal( 'test', false );
+			$this->marketingFooter = $this->request->getVal( 'marketingFooter' );
```

Comment:
If no `marketingFooter` is passed in this will return `null`, we probably want a boolean here yea?

What about making `$marketingFooter = false;` on L17 a class constant like 
```php
const MARKETING_FOOTER_DEFAULT = false
``` 
and changing this line to 
```php
$this->marketingFooter = $this->request->getVal( 'marketingFooter', self::MARKETING_FOOTER_DEFAULT );
```